### PR TITLE
docs: fix css-extract-rspack-plugin typos

### PR DESCRIPTION
**Pull request by edbzn**
## Summary

Fix typo in new css-extract-rspack-plugin en doc.

## Checklist

- [x] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/en/plugins/rspack/css-extract-rspack-plugin.mdx [hunk 1]
```diff
@@ -160,7 +160,7 @@ module.exports = {
     rules:[
       {
         test: /\.css$/i,
-        use: [rspack.CssExtractRspacKplugin.loader,'Css-Loader']
+        use: [rspack.CssExtractRspackPlugin.loader,'css-Loader']
```
Comment (xc2) [inline website/docs/en/plugins/rspack/css-extract-rspack-plugin.mdx:163]: thank you! could you please also help changing `css-Loader` to `css-loader`

**Review by xc2 — CHANGES_REQUESTED** [review]

File: website/docs/en/plugins/rspack/css-extract-rspack-plugin.mdx (hunk 1 above)
Comment (ahabhgk) [inline website/docs/en/plugins/rspack/css-extract-rspack-plugin.mdx:163]:
```suggestion
        use: [rspack.CssExtractRspackPlugin.loader,'css-loader']
```

**Review by ahabhgk — APPROVED** [review]

**Comment by xc2** [comment]
> [CLA assistant check](https://cla-assistant.io/web-infra-dev/rspack?pullRequest=6184) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/web-infra-dev/rspack?pullRequest=6184) before we can accept your contribution.**1** out of **2** committers have signed the CLA.✅ ahabhgk❌ edbznYou have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/web-infra-dev/rspack?pullRequest=6184) it.

@edbzn could you please sign the CLA

**Comment by chenjiahan** [comment]
Thanks ❤️

**Pull request merged**